README.md

```diff
@@ -205,6 +205,18 @@ Run blazepack in verbose mode and share the log by raising an issue
 ```
 blazepack start --verbose > blazepack.log
 ```
+### Contributing To This Project
+
+```
+Fork this repository to make your own copy
```

Comment:
add only the commands inside the backticks and keep other content out of it for better readability